packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -84,121 +85,125 @@ module.exports = {
     // We also include JSX as a common component filename extension to support
     // some tools, although we do not recommend using it, see:
     // https://github.com/facebookincubator/create-react-app/issues/290
-    extensions: ['.js', '.json', '.jsx', ''],
+    extensions: ['.js', '.json', '.jsx', '.scss', ''],
     alias: {
       // Support React Native Web
       // https://www.smashingmagazine.com/2016/08/a-glimpse-into-the-future-with-react-native-for-web/
-      'react-native': 'react-native-web'
+      'react-native': 'react-native-web',
+      // Access our required modules with a short alias
+      component: path.resolve('./node_modules/@lostmyname/components/dist'),
+      chameleon: path.resolve('./node_modules/chameleon-sass/assets/stylesheets'),
+      css: path.resolve('./node_modules/@lostmyname/css/scss'),
```

Comment:
I think we should alias to:
@lostmyname/components
@lostmyname/css

Just to be clear.